Description: fix functional test diff_patch_drop
 Old GNU patch versions were a bit aggressive and removed an empty directory,
 where they should have removed only the file. Let's not rely on that and
 allow the test to pass with newer GNU patch versions that behave better.
Origin: upstream, commit: f79d8215061cd57a97ecfe3bd02a7748ff605cf5
Author: Markus Wanner <markus@bluegap.ch>
--- a/test/func/diff_patch_drop/__driver__.lua
+++ b/test/func/diff_patch_drop/__driver__.lua
@@ -32,9 +32,10 @@
     check({"patch", "-p0", "-E"}, 0, false, false, true)
     check(not exists("dir/file"))
 else
-    -- GNU patch should only remove the file
-    -- but it is a little aggressive and removes the dir too!
     check({"patch", "-p0"}, 0, false, false, true)
     check(not exists("dir/file"))
-    check(not exists("dir"))
+
+    -- GNU patch should only remove the file but older versions may be a
+    -- little aggressive and remove the dir too! We test monotone, here,
+    -- so we shouldn't care whether or not "dir" still exists.
 end
